<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChosenMovie</class>
 <widget class="QWidget" name="ChosenMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="CriteriaLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="YearSpace"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="GenreSpace"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="ScoreSpace"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Runtime</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="RuntimeSpace"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Director</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="DirectorSpace"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <layout class="QVBoxLayout" name="MovieLayout">
     <item>
      <layout class="QHBoxLayout" name="MovieInfoLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="MovieDescriptionLayout">
       <item>
        <layout class="QVBoxLayout" name="VerticalInfoLayout"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="DescriptionLayout">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QTextBrowser" name="DescriptionSpace">
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="MovieButtonsLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="CopyButton">
         <property name="text">
          <string>Copy name and year</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="DownloadButton">
         <property name="text">
          <string>Download info to txt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="RandomGenerator">
     <property name="text">
      <string>Generate New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
